tests/test_revert.py

```diff
@@ -0,0 +1,305 @@
+import pytest
+import brownie
+from brownie import accounts, Contract, HomoraIBSwap
+
+SAFEBOX_ABI = [
```

Comment:
This should not be needed. Can you use `interfaces.ISafeBox.at(...)` instead of `Contract.from_abi`?